src/pages/students/StudentsOverview.jsx

```diff
@@ -1,7 +1,141 @@
+const columns = [
+  { id: 'id', label: 'ID', align: 'left' },
+  { id: 'code', label: 'Code', align: 'right' },
+  { id: 'name', label: 'Name', align: 'right' },
+  { id: 'email', label: 'Email', align: 'right' },
+  { id: 'phone', label: 'Phone', align: 'right' },
+  { id: 'status', label: 'Status', align: 'right' },
+  { id: 'createdAt', label: 'CreatedAt', align: 'right' },
+  { id: 'updatedAt', label: 'UpdatedAt', align: 'right' }
+];
+
 const StudentsOverview = () => {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+  const [data, setData] = useState([]);
+  const [info, setInfo] = useState({});
+
+  const token = localStorage.getItem('token');
+  useEffect(() => {
+    axios
+      .get(baseURL, {
+        headers: {
+          Authorization: `Bearer ${token}`
+        }
+      })
+      .then((res) => {
+        setData(res.data.data);
+        setInfo(res.data.info);
+        console.log(res.data);
+        const tableBody = document.getElementById('table-body');
+        if (tableBody) {
+          let html = '';
+          for (let i = 0; i < data.length; i++) {
+            console.log(
+              `ID: ${data[i].id}, Code: ${data[i].code}, Name: ${data[i].name}, Email: ${data[i].email}, Phone: ${data[i].phone}`
+            );
+            html += `<tr>
+                    <td>${data[i].id}</td>
+                    <td>${data[i].code}</td>
+                    <td>${data[i].name}</td>
+                    <td>${data[i].email}</td>
+                    <td>${data[i].phone}</td>
+                </tr>`;
+          }
+          tableBody.innerHTML = html;
+        }
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, [token]);
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
   return (
-    <div>
-      <h1>StudentsOverview</h1>
+    <div className="OverviewStudent">
+      <div className="sideBar">
+        <Sidebar />
+      </div>
+      <div className="ViewStudent">
+        <div className="BodyOveridetudent">
+          <div className="OverideHeaderStudent">
+            <h1 className="Title">Register Student</h1>
+          </div>
+          <div className="TableCenter">
+            <TableContainer component={Paper}>
+              <Table sx={{ minWidth: 650 }} size="small" aria-label="a dense table">
+                <TableHead>
+                  <TableRow>
+                    {columns.map((column) => (
+                      <TableCell key={column.id} align={column.align}>
+                        {column.label}
+                      </TableCell>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => (
```

Comment:
Backend already do the pagination, so we have to manage this values on the request

Reply:
ok teacher

Reply:
listo profe